### ISSUE TITLE
Update App Headings for Accessibility

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7817]
# Project Status Overview

### Project Objective:
We completed an audit of the app headings found in the mobile app in January of 2024. Those results can be found [in this Google sheet](https://docs.google.com/spreadsheets/d/1KnR9Isdw8e3aBp5WhOFk7vaR4p6v4-wpYIO4GsEk4Go/edit?usp=sharing). As a result of those findings and our internal meeting on January 18, it was determined that an epic should be created so that we could break the necessary updates down by screens / areas within the app.
